Gamma_Analysis.py

```diff
@@ -418,12 +420,15 @@ def check_spectra(samples, background, reference):
                             error_spectrum)
 
 
-def main():
-    background = SPEFile.SPEFile("USS_Independence_Background.Spe")
+def main(background_filename=BACKGROUND, reference_filename=REFERENCE, 
+         file_list=None):
+    
+    background = SPEFile.SPEFile(background_filename)
     background.read()
-    reference = SPEFile.SPEFile("UCB018_Soil_Sample010_2.Spe")
+    reference = SPEFile.SPEFile(reference_filename)
     reference.read()
-    sample_measurements, sample_names = acquire_files()
+    if file_list is None:
+        sample_measurements, sample_names = acquire_files()
```

Comment:
I meant that you write that function. It would just perform the `sample_names = [...` assignment from `acquire_files`. Then `acquire_files` could call `get_sample_names` and you could also call `get_sample_names` in the case that a `file_list` is provided in `main`.